xml/System.Dynamic/DynamicObject.xml

```diff
@@ -87,7 +87,8 @@ Dim sampleObject As Object = New SampleDynamicObject()
  [!code-vb[System.Dynamic.DynamicObject#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.dynamic.dynamicobject/vb/module1.vb#1)]  
   
  For more examples, see [Creating Wrappers with DynamicObject](https://go.microsoft.com/fwlink/?LinkId=169008) on the C# Frequently Asked Questions blog.  
```

Comment:
This should change to https://devblogs.microsoft.com/csharpfaq/dynamic-in-c-4-0-creating-wrappers-with-dynamicobject/

Do you want to make this change @mvelosop ?

Reply:
Thanks @BillWagner, I'll do it 😊